src/core/types/api/data-sync.ts

```
@@ -0,0 +1,873 @@
+/**
+ * Convert `set` and `remove` parameters to JSON Patch operations (wire format).
+ *
+ * - Each key in `set` becomes a "replace" operation.
+ * - Each entry in `remove` becomes a "remove" operation.
+ *
+ * @internal
+ */
+export function toJsonPatchOperations(
+  set?: Record<string, unknown>,
+  remove?: string[],
+): JsonPatchOperation[] {
```

ESLint

[error] Replace `⏎··set?:·Record<string,·unknown>,⏎··remove?:·string[],⏎` with `set?:·Record<string,·unknown>,·remove?:·string[]`

(prettier/prettier)